frontend/pages/club/[club]/flyer.js

```diff
@@ -9,8 +9,7 @@ import {
   Center,
   Metadata,
 } from '../../../components/common'
-import { CLUBS_BLUE, CLUBS_NAVY, CLUBS_RED } from '../../../constants/colors'
-import ClubMetadata from '../../../components/ClubMetadata'
```

Comment:
@cacabo Do you think it would be better to use ClubMetadata or Metadata here, or should it not matter as much since the flyer pages aren't accessible from any path of pages from index?

Reply:
Ideally ClubMetadata but with an additional change to the title